src/app/background/sagas/refreshContributors.ts

```diff
@@ -8,6 +8,7 @@ export function* refreshContributorsPeriodicallySaga() {
     Number(process.env.REFRESH_CONTRIBUTORS_INTERVAL)
   );
 
+  // warning SSR: refresh LOOP
```

Comment:
Hum, `background` app is supposed to be the "heart" of the extension this shouldn't be included in the SSR process meaning:
- A user without the extension go to the profile app, all infos are supposed to come from the API not background.
- A user has the extension installed some of informations are fetched from background (ex: subscriptions)